runtime.md

```diff
@@ -9,8 +9,8 @@ $ funC [global-options] <COMMAND> [command-specific-options] <command-specific-a
 
 ## Global options
 
-None are required, but the runtime may support options that start with at least one hyphen.
-Global options may take positional arguments (e.g. `--log-level debug`), but the option parsing must be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`), but the option parsing MUST be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` for any `<COMMAND>` that does not start with a hyphen (including commands not specified in this document).
 
```

Comment:
On Fri, Dec 11, 2015 at 08:03:31AM -0800, Mike Brown wrote:
> Reviewed looks good, with note to PR #8 for inclusion after this drops. 

#8 is earlier in the queue, so I just landed it.  I'll rebase this PR
on top.